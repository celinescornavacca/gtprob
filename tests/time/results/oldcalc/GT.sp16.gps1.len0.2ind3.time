0.40user 0.13system 0:00.68elapsed 79%CPU (0avgtext+0avgdata 29728maxresident)k
125inputs+0outputs (0major+12612minor)pagefaults 0swaps
