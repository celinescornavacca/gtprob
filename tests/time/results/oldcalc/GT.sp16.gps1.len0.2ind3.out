-13.938180138
